quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/StoragePropertyMappers.java

```diff
@@ -303,6 +303,10 @@ public static PropertyMapper[] getMappers() {
                         .to("kc.spi-map-storage-file-dir")
                         .mapFrom("storage")
                         .paramLabel("dir")
+                        .build(),
+                fromOption(StorageOptions.STORAGE_JPA_DB)
+                        .mapFrom("storage")
```

Comment:
We could also map the property to some internal `spi` one as well, right?

Reply:
I'm not sure what `spi` you have in mind here, could you please elaborate on this?

Reply:
I mean keep the convention around mapping these properties to some with prefix `kc.spi-`.
```suggestion
                        .to("kc.spi-map-storage-jpa-db")
                        .mapFrom("storage")
```

Reply:
Updated